packages/wrangler/src/config/index.ts

```diff
@@ -670,7 +670,11 @@ function tryLoadDotEnv(path: string): DotEnv | undefined {
 		const parsed = dotenv.parse(fs.readFileSync(path));
 		return { path, parsed };
 	} catch (e) {
-		logger.debug(`Failed to load .env file "${path}":`, e);
+		if ((e as { code: string }).code === "ENOENT") {
+			logger.debug(`.env file not found at "${path}"`);
```

Comment:
definitely less scary but it still could be mistaken for an error - can we add a short message to say this is fine to ignore if you're not trying to use .dev.vars, .env etc.?